### PR TITLE
Telemax Robot

### PR DESCRIPTION
**Pull request by DavidMansolino**
This PR adds a model of the Telemax robot from Telerob.

Remaining tasks:
  - [x] Camera model.
  - [x] Camera slots.
  - [x] Track geometry model.
  - [x] Splash screen.
  - [x] Wheels
  - [x] World
  - [x] Demo controller
    - [x] Add camera zoom.
  - [x] Calibration:
    - [x] Masses
    - [x] Joint limits
    - [x] Motor torque/speed/acceleration
    - [x] Camera resolution/fov.
    - [x] Camera lights
  - [x] Docs: https://www.cyberbotics.com/doc/guide/telemax-pro?version=feature-telemax-robot
    - [x] Update final video link.

### CODE REVIEW

**Comment by DavidMansolino** [comment]
Progress with the track:
telemax15_2
ezgif-1-88de0ab067de

**Comment by DavidMansolino** [comment]
hall_2

**Comment by DavidMansolino** [comment]
With real track geometries instead of boxes:
telemax15_track

I will try to find a tradeoff between the two.

**Comment by DavidMansolino** [comment]
With the optionnal cameras (I managed to reduce the number of triangles of the articulated camera from 200'000 to ~3'000):

telemax15

**Comment by fabienrohrer** [comment]
This is for sure the most good-looking model ever done in Webots :heart:

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
https://youtu.be/vU086Fd22-0

**Comment by DavidMansolino** [comment]
test me

File: change_logs/ChangeLog.html [hunk 1]
```diff
@@ -17,6 +17,7 @@ <h2><a href='https://cyberbotics.com/doc/blog/Webots-2018-b-release'>Webots R201
 <ul>
 <li>Added an ABB IRB 4600/40 arm model.</li>
 <li>Added a Clearpath Moose model.</li>
+<li>Added a Telerob Telemax model.</li>
```
Comment (omichel) [inline change_logs/ChangeLog.html]:
```suggestion
<li>Added a Telerob Telemax PRO model.</li>
```

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
Remains only to calibrate the camera :smiley:

**Comment by DavidMansolino** [comment]
With controlled wheels:

ezgif-4-7d4509f50524

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
:+1:

**Comment by DavidMansolino** [comment]
:-1:  (I did forget the thumbnails)

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
:+1:

File: projects/samples/environments/factory/worlds/hall.wbt [hunk 2]
```diff
@@ -29,38 +29,44 @@ Floor {
...
+        ]
+        texCoordIndex [
+          0, 2, 3, 1, -1
+        ]
+      }
     }
-    metalness 0
-  }
+  ]
+  name "floor(1)"
```
Comment (omichel) [inline projects/samples/environments/factory/worlds/hall.wbt]:
```suggestion
  name "floor"
```

File: docs/guide/appearances.md [hunk 3]
```diff
@@ -37,6 +37,39 @@ Asphalt {
...
+
+```
+BakelitePlastic {
+  SFNode  textureTransform NULL
+  SFNode  environmentMap   NULL
+  SFFloat IBLStrength      1
+}
+```
+
+> **File location**: "WEBOTS\_HOME/projects/appearances/protos/BakelitePlastic.proto"
```
[the remark this answers is not in the document]
Comment (DavidMansolino) [reply docs/guide/appearances.md]: I will regenerate the doc, it should fix the problem for all the appearances.

File: projects/samples/environments/factory/worlds/hall.wbt (hunk 2 above)
Comment (DavidMansolino) [reply projects/samples/environments/factory/worlds/hall.wbt]: That's not going to be possible as the `name` of the `Floor` PROTO is `floor` too, but actually what I can do is to transform this `Solid` into a `Transform` to get ride of this field.

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
:+1:

**Pull request merged**